config/content/sass.json

```diff
@@ -9,6 +9,5 @@
         "xlarge": "1200px",
         "tablet": "760px",
         "desktop": "980px"
-    },
-    "themeColour": "#ec008c"
```

Comment:
This is being moved into the general config as we don't use it anywhere in our sass files so doesn't really need to be shared.